graylog2-server/src/main/java/org/graylog2/configuration/ElasticsearchConfiguration.java

```diff
@@ -17,26 +17,19 @@
 package org.graylog2.configuration;
 
 import com.github.joschi.jadconfig.Parameter;
-import com.github.joschi.jadconfig.ValidationException;
-import com.github.joschi.jadconfig.ValidatorMethod;
-import com.github.joschi.jadconfig.converters.StringListConverter;
 import com.github.joschi.jadconfig.util.Duration;
-import com.github.joschi.jadconfig.validators.FilePathReadableValidator;
-import com.github.joschi.jadconfig.validators.InetPortValidator;
 import com.github.joschi.jadconfig.validators.PositiveDurationValidator;
 import com.github.joschi.jadconfig.validators.PositiveIntegerValidator;
 import com.github.joschi.jadconfig.validators.PositiveLongValidator;
+import com.github.joschi.jadconfig.validators.StringNotBlankValidator;
 import org.joda.time.Period;
 
-import javax.validation.constraints.NotNull;
-import java.nio.file.Files;
-import java.nio.file.Path;
-import java.nio.file.Paths;
-import java.util.Collections;
-import java.util.List;
 import java.util.Locale;
 
 public class ElasticsearchConfiguration {
+    public static final String DEFAULT_EVENTS_INDEX_PREFIX = "default_events_index_prefix";
```

Comment:
Should we add these into our `graylog.conf`?

Reply:
I am a bit on the fence on this. This should **only** be used by users if they already have an index prefix of `gl-events` configured for one of their index sets. Adding it to the example `graylog.conf` might motivate users to change it. :wink:

Reply:
fair enough :)